capabilities/istio/.github/zarf-runner/bob-the-builder/zarf.yaml

```diff
@@ -25,7 +25,9 @@ components:
             cmd: |
               npm ci
               npx pepr build
-            dir: ../../../pepr
+            dir: ../../../../../
+            env:
+              - "CAPABILITY=istio"
           - description: Build the zarf packages
             cmd: |
-              ./zarf p c ../../../ --confirm --no-progress -o ../build
```

Comment:
I know you dont like the no progress because its automatic in ci, but if we are running the same in local dev and ci i dont think anyone actually wants the full output from local. 

Alternatively to having the --no-progress all over, maybe we can export CI=true as a first steps in the runner package? With the task runner stuff jordan is working on this would be even easier it could just be a global var at the top of the file.

Reply:
The output is mostly the same with no-progress or not on a regular terminal, it just turns off things that repost to the same line. It was created for streaming terminal recorders to not show the progress or spinner over and over. If you want to reduce logging (which I see as an individual decision), that would be done by changing the log level.